pkg/sdk/stages_dto_builders_gen.go

```diff
@@ -0,0 +1,825 @@
+// Code generated by dto builder generator; DO NOT EDIT.
+
+package sdk
+
+import ()
+
+func NewCreateInternalStageRequest(
+	name SchemaObjectIdentifier,
+) *CreateInternalStageRequest {
+	s := CreateInternalStageRequest{}
+	s.name = name
+	return &s
+}
+
+func (s *CreateInternalStageRequest) WithOrReplace(OrReplace *bool) *CreateInternalStageRequest {
+	s.OrReplace = OrReplace
+	return s
+}
+
+func (s *CreateInternalStageRequest) WithTemporary(Temporary *bool) *CreateInternalStageRequest {
+	s.Temporary = Temporary
+	return s
+}
+
+func (s *CreateInternalStageRequest) WithIfNotExists(IfNotExists *bool) *CreateInternalStageRequest {
+	s.IfNotExists = IfNotExists
+	return s
+}
+
+func (s *CreateInternalStageRequest) WithEncryption(Encryption *InternalStageEncryptionRequest) *CreateInternalStageRequest {
```

Comment:
It is an enum type, It's in the definition file `InternalStageEncryptionOption`. It's just that the generator generates a request type for this enum, so instead of `.WithEncryption(myEnum)` it `.WithEncryption(MyEnumRequest(myEnum))`. I think we can leave it as is and this will change after dto builder refactoring. I'll add note to the readme about enums.